proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java

```diff
@@ -219,7 +217,7 @@ public void done(ServerPing result, Throwable error)
                     public void done(ProxyPingEvent pingResult, Throwable error)
                     {
                         Gson gson = BungeeCord.getInstance().gson;
-                        unsafe.sendPacket( new StatusResponse( gson.toJson( pingResult.getResponse() ) ) );
```

Comment:
unsafe is for plugins. Theory it does not matter. But its implementation, and I decided to use channel class